addons/web/static/tests/webclient/helpers.js

```diff
@@ -406,3 +407,9 @@ export function getActionManagerServerData() {
         menus,
     };
 }
+
+export async function setBrowserLocation(value) {
+    Object.assign(browser.location, value);
+    // hashchange event isn't trigerred synchronously, so we have to wait for it
+    await nextTick(); // wait for the hashchange event
```

Comment:
comments are redundant

Reply:
I would move this function to test/helpers/utils.js (not really webclient related)